### PR TITLE
Add more elfinder typing

### PR DESCRIPTION
**Pull request by MartinHjelmare**

### CODE REVIEW

File: jupyter_elfinder/elfinder.py [hunk 1]
```diff
@@ -990,24 +1043,28 @@ def __info(self, path):
...
 
-    def __tree(self, path, depth=0):
+    def __tree(self, path: str, depth: int = 0) -> Dict[str, Any]:
         """Return directory tree starting from path."""
 
         if not os.path.isdir(path):
-            return ""
+            # pylint: disable=fixme
+            # TODO: Is it ok to return an empty dict instead of empty string
+            return {}
```
Comment (MartinHjelmare) [inline jupyter_elfinder/elfinder.py]: Is this change ok?

File: jupyter_elfinder/elfinder.py [hunk 2]
```diff
@@ -1398,7 +1491,7 @@ def __rm_tmb(self, path):
                 except OSError:
                     pass
 
-    def __readlink(self, path):
+    def __readlink(self, path: str) -> Optional[str]:
```
Comment (oeway) [inline jupyter_elfinder/elfinder.py]: Change to __read_link ?

File: jupyter_elfinder/elfinder.py (hunk 1 above)
Comment (oeway) [reply jupyter_elfinder/elfinder.py]: Looks good.

File: jupyter_elfinder/elfinder.py (hunk 2 above)
Comment (MartinHjelmare) [reply jupyter_elfinder/elfinder.py]: Yes.

File: jupyter_elfinder/elfinder.py (hunk 1 above)
Comment (MartinHjelmare) [reply jupyter_elfinder/elfinder.py]: Ok, thanks.

**Pull request merged**